### DISCUSSION TITLE
[New Component] [watermark ] 官方是否添加水印功能 可以通过v-model 控制打开和关闭

### DISCUSSION THREAD

**Discussion by Liang-Dongxing** [element-plus/element-plus#10194] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `el-watermark `

## Description

官方是否添加水印功能

### Additional comments

_(empty)_

**Comment by applekc** [comment]
ant-design推出了WaterMark水印组件：https://ng.ant.design/components/water-mark/zh

**Comment by chenxch** [comment]
follow #14236
